source/install/install-ubuntu-1604-mattermost.rst

```diff
@@ -84,13 +84,16 @@ Assume that the IP address of this server is 10.10.10.2.
     LimitNOFILE=49152
 
     [Install]
-    WantedBy=multi-user.target
+    WantedBy=postgresql.service
```

Comment:
This only works for running the DB server on the same machine as the Mattermost app server. Which is not the recommended configuration. Let's have this as the generic with a note at the bottom to change this if you are running the MM server on the same machine as the DB.

Reply:
@crspeller: the section of the documentation where I'm suggesting changes defaults to them running on the same machine, and rather notes what you should to different *if* you are running the database on a dedicated server:

> **Note**
> If you have installed MySQL or PostgreSQL on a dedicated server then you need to remove the `After=postgresql.service` and `Requires=postgresql.service` or `After=mysql.service` and `Requires=mysql.service` lines in the `[Unit]` section or the Mattermost service will not start.

Are you suggesting we keep it `WantedBy=multi-user.target`, and instead remove both`After=postgresql.service` and `Requires=postgresql.service` and flip the notes?

Reply:
Ahh. Sorry I assumed our default would be a multi machine setup. This is fine then.